bears/general/LineBreakBear.py

```diff
@@ -0,0 +1,133 @@
+from queue import Queue
+
+from coalib.results.AbsolutePosition import AbsolutePosition
+from coalib.results.Diff import Diff
+from coalib.bearlib.languages.LanguageDefinition import LanguageDefinition
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result, RESULT_SEVERITY
+from coalib.results.SourceRange import SourceRange
+from coalib.bearlib.spacing.SpacingHelper import SpacingHelper
+
+from bears.general.AnnotationBear import AnnotationBear
+from bears.general.RangeHelpers import get_specified_block_range
```

Comment:
also, everything else in that directory is a bear, while `RangeHelpers` is most likely not, right? would it make more sense to put that functionality somewhere else? maybe eben `coala/coalib/bearlib/...` or similar?